packages/@tailwindcss-cli/src/commands/build/index.ts

```diff
@@ -95,6 +95,15 @@ export async function handle(args: Result<ReturnType<typeof options>>) {
     }
   }
 
+  // Check if the input and output file paths are identical, otherwise return an
+  // error to the user.
+  if (args['--input'] === args['--output']) {
```

Comment:
Before we merge this I want to check 1 small thing. Commenting here to prevent auto merge from happening.

Reply:
So one edge case here is that `-` can be used by both the `--input` and `--output`, e.g.: `--input - --output -`. The `-` means "use stdin and stdout respectively". In this case we don't want the error to show up.

Added a failing integration test + a fix for this.

(I also ran prettier in a separate commit)